cmd/initialize/prepare.go

```diff
@@ -0,0 +1,74 @@
+package initialize
+
+import (
+	"fmt"
+	"log/slog"
+	"os"
+	"strings"
+
+	"numerous.com/cli/cmd/initialize/wizard"
+	"numerous.com/cli/cmd/output"
+	"numerous.com/cli/internal/dir"
+	"numerous.com/cli/internal/manifest"
+	"numerous.com/cli/internal/python"
+)
+
+func PrepareInit(args []string) (string, *manifest.Manifest, error) {
+	appDir, err := os.Getwd()
+	if err != nil {
+		slog.Info("An error occurred when trying to get the current user path during init process.", slog.String("error", err.Error()))
+		fmt.Println(err)
+
+		return "", nil, ErrGetWorkDir
+	}
+
+	if len(args) != 0 {
+		appDir = PathArgumentHandler(args[0], appDir)
+	}
+
+	if exist, _ := dir.AppIDExists(appDir); exist {
```

Comment:
should this be dir.tomlExist or something along those lines, as AppID is being phased out?

Reply:
Yes, changed to ManifestExists